src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -606,6 +643,12 @@ ConvSolution InitInvokerFactoryNCHW(const ExecutionContext& ctx,
                 std::swap(conv_tensors.xDesc, conv_tensors.yDesc);
             }
 
+            HipEventPtr start = nullptr;
+            HipEventPtr stop  = nullptr;
+            if(handle.IsProfilingEnabled())
+            {
+                ProfilingRecordStart(handle, start, stop);
+            }
```

Comment:
At line 660 below there is:
https://github.com/ROCm/MIOpen/blob/81f29c00af6dcc3e9053390953373dc55dc45117/src/include/miopen/solver/implicitgemm_ck_util.hpp#L658-L663
Please delete "Fix NHWC Wrw invokers to also issue a zero-out kernel." from the todo comment. The second statement is still needed - it is about of replacing ZeroOutBuffer() with SetTensor() AFAICS.

/cc @amberhassaan

Reply:
Of course I am not against replacing ZeroOutBuffer() with SetTensor() right here, but this is a bit out of scope of this PR, so I am not insisting on anything. Up to you.